### PR TITLE
feat: add policy tag manager clients

### PR DESCRIPTION
**Pull request by tswast**
Generated by synthtool.

Addresses internal issue 138402054.

### CODE REVIEW

**Comment by bcoe** [comment]
@tswast this looks like it's potentially an issue with the protos themselves, perhaps a missing import?

```
  1) PolicyTagManagerClient
       updateTaxonomy
         invokes updateTaxonomy without error:
     TypeError: Cannot read property 'name' of undefined
      at PolicyTagManagerClient.updateTaxonomy (src/v1beta1/policy_tag_manager_client.js:392:41)
      at Context.<anonymous> (test/gapic-v1beta1.js:1878:14)
      at processImmediate (internal/timers.js:439:21)

  2) PolicyTagManagerClient
       updateTaxonomy
         invokes updateTaxonomy with error:
     TypeError: Cannot read property 'name' of undefined
      at PolicyTagManagerClient.updateTaxonomy (src/v1beta1/policy_tag_manager_client.js:392:41)
      at Context.<anonymous> (test/gapic-v1beta1.js:1901:14)
      at processImmediate (internal/timers.js:439:21)```

**Comment by tswast** [comment]
Looks like a test that didn't actually specify an input value to me. Maybe the generator has a bug? Or is there something we need to add to the gapic config?

**Comment by bcoe** [comment]
@alexander-fenster any thoughts on this one? Perhaps we need to move this to the new generator sooner rather than later.

**Comment by JustinBeckwith** [comment]
I am closing this, and hoping that we'll wake up to a better PR in the morning 😆

**Comment by tswast** [comment]
> I am closing this, and hoping that we'll wake up to a better PR in the morning 😆

Unfortunately we need to make an update to the synth config before we get a real PR for this, because the config file moved directories slightly.

**Comment by tswast** [comment]
Sent https://github.com/googleapis/nodejs-datacatalog/pull/59

File: src/v1beta1/policy_tag_manager_client.js [hunk 1]
```diff
@@ -0,0 +1,982 @@
...
+      options = {};
+    }
+    request = request || {};
+    options = options || {};
+    options.otherArgs = options.otherArgs || {};
+    options.otherArgs.headers = options.otherArgs.headers || {};
+    options.otherArgs.headers[
+      'x-goog-request-params'
+    ] = gax.routingHeader.fromParams({
+      'policy_tag.name': request.policyTag.name,
```
Comment (tswast) [inline src/v1beta1/policy_tag_manager_client.js]: I see this reads from `policyTag.name`, but it's not showing as a parameter in the function's jsdoc.

File: protos/google/cloud/datacatalog/v1beta1/policytagmanager.proto [hunk 2]
```diff
@@ -0,0 +1,420 @@
...
+// "City", and "ZipCode".
+message PolicyTag {
+  option (google.api.resource) = {
+    type: "datacatalog.googleapis.com/PolicyTag"
+    pattern: "projects/{project}/locations/{location}/taxonomies/{taxonomy}/policyTags/{policy_tag}"
+  };
+
+  // Output only. Resource name of the policy tag, whose format is:
+  // "projects/{project_number}/locations/{location_id}/taxonomies/{taxonomy_id}/policyTags/{id}".
+  string name = 1 [(google.api.field_behavior) = OUTPUT_ONLY];
```
Comment (tswast) [inline protos/google/cloud/datacatalog/v1beta1/policytagmanager.proto]: It appears that name is present in the relevant proto, though.

**Pull request closed without merging**